simple_queue.py

```diff
@@ -0,0 +1,107 @@
+import json
+import cv2
+
+from settings import RABBITMQ_PASSWORD, RABBITMQ_LOGIN, RABBITMQ_SERVER, RABBITMQ_EXCHANGE_NAME
+from net_tools import send_json, get_image_from_token, load_schema
+
+from typing import Dict, List
+from pika import ConnectionParameters, PlainCredentials, BlockingConnection
+
+credentials = PlainCredentials(RABBITMQ_LOGIN, RABBITMQ_PASSWORD)
+connection_parameters = ConnectionParameters(host=RABBITMQ_SERVER, port=5672, virtual_host="/", credentials=credentials)
+exchange_name = RABBITMQ_EXCHANGE_NAME
+
+
+def processResponse(response, method_frame):
+
+    data = json.loads(response.decode('utf-8'))
+    print(f"Received message : {data}")
+    print("[INFO] Downloading image.")
```

Comment:
I would rather go for [logging](https://docs.python.org/3/library/logging.html) module here

```python
import logging
logging.info("Downloading image")
```